wandb/sdk/internal/system/assets/open_metrics.py

```diff
@@ -106,6 +156,14 @@ def parse_open_metrics_endpoint(self) -> Dict[str, Union[str, int, float]]:
                 continue
             for sample in family.samples:
                 name, labels, value = sample.name, sample.labels, sample.value
+
+                if not _should_capture_metric(
+                    name,
+                    tuple(labels.items()),
+                    _nested_dict_to_tuple(self.filters),
```

Comment:
compute _nested_dict to tuple when filters are set, def not inside this loop of loops

Reply:
good catch, fixed!